src/k8s-engine/lima.ts

```diff
@@ -245,6 +245,8 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
 
   debug = false;
 
+  emit: K8s.KubernetesBackend['emit'] = this.emit;
```

Comment:
This changes the type of `emit` from `EventEmitter['emit']` (i.e. `emit(event: string | symbol, ...args: any[]): boolean;`) to our type, so that we check the event name against our whitelist.  We don't need this for `on()`, etc. because the callers only get a `KubernetesBackend`, not a <thing extending `EventEmitter`>.